code/API_definitions/sim-swap.yaml

```
@@ -73,13 +77,13 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: 1.0.0
+  version: wip
   x-camara-commonalities: 0.4.0
 externalDocs:
   description: Product documentation at Camara
   url: https://github.com/camaraproject/SimSwap
 servers:
-  - url: "{apiRoot}/sim-swap/v1"
+  - url: "{apiRoot}/sim-swap/wip"
```

@bigludo7  shouldn't this be vwip?

---

@fernandopradocabrillo Yes - this is strange because I've fixed this one in https://github.com/camaraproject/SimSwap/pull/153/commits/7acbc2c4caba695cecf460fb1322dfe2bc7e5c2e . Could you check if it's good now? Thanks

---

In that commit I also see it as wip. Let me place the suggestion so we can apply directly

```suggestion
  - url: "{apiRoot}/sim-swap/vwip"
```